docs/pages/desktop-access/active-directory-manual.mdx

```diff
@@ -682,6 +682,49 @@ Computer Configuration > Policies > Windows Settings > Security Settings > Local
 Double click the "Add workstations to domain" policy and ensure that the
 "Authenticated Users" group is not present.
 
+## Multiple Domains
```

Comment:
Reminder to self: it probably makes sense to mention the version that this will be released in.